modules/ROOT/pages/index.adoc

```diff
@@ -20,6 +20,66 @@ With Mule you can:
 * Use error constructs and try scopes for rapid debugging.
 * Isolate the classloader to protect Mule apps from changes to the runtime or connectors.
 
+[[install_deploy_manage]]
+== Installation, Deployment, Management
+
+//TODO? THIS BEGS FOR A DIAGRAM TO SHOW THE RELATIONSHIPS DESCRIBED HERE
+
+For a Mule app to run, it must be deployed into an environment where the Mule
```

Comment:
"The Mule": I find it confusing when I read it. I see that at the beginning of the document you start with "Mule runtime engine (Mule)" so it is expected from the reader to do the mapping, but still is an extra mental step that is hard to make when you are learning the concept. Also, there are places where it says "The mule" and others where you use just "Mule"

Reply:
thx! changed to "the Mule runtime engine"